test/functional/world_location_news_controller_test.rb

```diff
@@ -21,17 +21,17 @@ def default_search_options
     }
   end
 
-  def announcement_search_options
+  def news_and_communications_search_options
     default_search_options.merge(
       count: 2,
-      filter_content_store_document_type: announcement_document_types
+      filter_content_purpose_supergroup: 'news_and_communications'
```

Comment:
Do we also need to filter on document type as we did previously, to ensure that only world items are being shown?

Reply:
apparently this shouldn't be an issue because the news and comms supergroup already has the world news type as part of the supergroup, as seen [here](https://github.com/alphagov/govuk_document_types/blob/master/data/supertypes.yml#L504)